presto-main/src/test/java/io/prestosql/sql/planner/planprinter/TestWindowOperatorStats.java

```diff
@@ -0,0 +1,26 @@
+package io.prestosql.sql.planner.planprinter;
+
+import io.prestosql.operator.WindowInfo;
+import org.testng.annotations.Test;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestWindowOperatorStats
+{
+    @Test
+    public void testWindowOperatorStatsForWindowInfoWithEmptyDriverInfosList()
+    {
+        WindowInfo info = new WindowInfo(emptyList());
+
+        WindowOperatorStats stats = WindowOperatorStats.create(info);
+
+        assertThat(stats.getIndexSizeStdDev()).isEqualTo(Double.NaN);
+        assertThat(stats.getIndexPositionsStdDev()).isEqualTo(Double.NaN);
+        assertThat(stats.getIndexCountPerDriverStdDev()).isEqualTo(Double.NaN);
+        assertThat(stats.getPartitionRowsStdDev()).isEqualTo(Double.NaN);
+        assertThat(stats.getRowsPerDriverStdDev()).isEqualTo(Double.NaN);
```

Comment:
How will these NaNs be printed in EXPLAIN ANALYZE?

Reply:
```
Active Drivers: [ 0 / 0 ]
Index size: std.dev.: ? bytes , ? rows
Index count per driver: std.dev.: ?
Rows per driver: std.dev.: ?
Size of partition: std.dev.: ?
```
That's how it will be handled right now